frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesList.cy.ts

```diff
@@ -0,0 +1,113 @@
+    },
+    mockRouteK8sResource({}),
+  );
+  cy.intercept(
+    { pathname: '/api/k8s/apis/kubeflow.org/v1/namespaces/test-project/notebooks' },
+    mockK8sResourceList([mockNotebookK8sResource({})]),
+  );
+  cy.intercept(
+    { pathname: '/api/k8s/apis/project.openshift.io/v1/projects' },
+    mockK8sResourceList([mockProjectK8sResource({})]),
+  );
+  cy.intercept(
+    { pathname: '/api/k8s/api/v1/namespaces/test-project/persistentvolumeclaims' },
+    mockK8sResourceList([mockPVCK8sResource({})]),
+  );
+  cy.intercept(
+    { pathname: '/api/k8s/apis/project.openshift.io/v1/projects/test-project' },
+    mockProjectK8sResource({}),
+  );
+  cy.intercept(
+    { pathname: '/api/k8s/api/v1/namespaces/test-project/secrets' },
+    mockK8sResourceList([mockSecretK8sResource({})]),
+  );
+  cy.intercept(
+    {
+      pathname: `/api/k8s/apis/route.openshift.io/v1/namespaces/test-project/routes/ds-pipeline-pipelines-definition`,
+    },
+    mockRouteK8sResource({
+      notebookName: 'ds-pipeline-pipelines-definition',
+    }),
+  );
+};
+
+describe('PipelinesList', () => {
+  it('should import button be disabled when the server is not configured', () => {
+    initIntercepts();
+    cy.intercept(
+      {
+        pathname: `/api/k8s/apis/datasciencepipelinesapplications.opendatahub.io/v1alpha1/namespaces/test-project/datasciencepipelinesapplications/pipelines-definition`,
+      },
+      {
+        statusCode: 404,
+        body: mock404Error({}),
+      },
+    );
+    projectDetails.visit('test-project');
+
+    cy.findByTestId('import-pipeline-split-button').should('be.disabled');
+  });
+
+  it('should import button be disabled when the server is initializing', () => {
+    initIntercepts();
+    cy.intercept(
+      {
+        pathname: `/api/k8s/apis/datasciencepipelinesapplications.opendatahub.io/v1alpha1/namespaces/test-project/datasciencepipelinesapplications/pipelines-definition`,
+      },
+      mockDataSciencePipelineApplicationK8sResource({ initializing: true }),
+    );
+    projectDetails.visit('test-project');
+
+    cy.findByTestId('import-pipeline-split-button').should('be.disabled');
+  });
+
+  it('should upload version button be disabled when the list is empty', () => {
+    initIntercepts();
+    cy.intercept(
+      {
+        pathname: `/api/k8s/apis/datasciencepipelinesapplications.opendatahub.io/v1alpha1/namespaces/test-project/datasciencepipelinesapplications/pipelines-definition`,
+      },
+      mockDataSciencePipelineApplicationK8sResource({}),
+    );
+    cy.intercept(
+      {
+        method: 'POST',
+        pathname: '/api/proxy/apis/v1beta1/pipelines',
+      },
+      buildMockPipelines([]),
+    );
+    projectDetails.visit('test-project');
+
+    cy.findByTestId('import-pipeline-split-button').should('be.enabled').click();
+    cy.get('#import-pipeline-version-button').should('have.attr', 'aria-disabled', 'true');
```

Comment:
Are we avoiding page objects and just selecting elements like this? 🤔 Any reason why?

Reply:
Hmm, I thought this test is simple enough so I don't need to create a page object. Yeah, I can do it definitely.